### PR TITLE
Major Repo Restructure

### PR DESCRIPTION
**Pull request by Uokoroafor**
Have restructured the repo to be in line with the grander project plan. 

There are now the three big pillars as originally envisioned:
1. API + backend
2. Frontend
3. Cloudformation

There are still a few enhancements to effect in the underlying API but I will weave them in as AWS is built up.

Thanks as always and of course no rush :)

### CODE REVIEW

File: application_frontend/__init__.py
Comment (sachahu1) [inline application_frontend/__init__.py:1]: No `__init__.py` for your frontend

File: cloudformation_architecture/__init__.py
Comment (sachahu1) [inline cloudformation_architecture/__init__.py:1]: no `__init__.py` for your infrastructure

File: pyproject.toml
Comment (sachahu1) [inline pyproject.toml:1]: Poetry files can move into your `file_manager` folder

**Comment by sachahu1** [comment]
Move/Remove `test_main.http`: https://github.com/Uokoroafor/file-transfer-api/blob/700cd8990ea4a6196b572e1bfb56c64aaf9e216a/test_main.http#L1C16-L1C16

**Comment by sachahu1** [comment]
I think you should remove then .gitignore your `.idea` folder

**Review by sachahu1 — COMMENTED** [review]
Overall looks good. Minor fixes

File: cloudformation_architecture/__init__.py
Comment (Uokoroafor) [reply cloudformation_architecture/__init__.py:1]: There was also one in the file_transfer_api which I've taken out. Good spot thanks!

File: pyproject.toml
Comment (Uokoroafor) [reply pyproject.toml:1]: Ah didn't realise you could have multiple toml files for one project. Everyday is a school day. Thanks.

Comment (sachahu1) [reply pyproject.toml:1]: Um... So, you can do that but I don't think you need to in this case?

**Pull request merged**